### PR TITLE
feat: added TWTG VB v4

### PR DESCRIPTION
**Pull request by martinichka**

### CODE REVIEW

File: vendors/twtg/drivers/neon-vb-v4/examples.json [hunk 1]
```diff
@@ -0,0 +1,936 @@
+[
```
Comment (martinichka) [inline vendors/twtg/drivers/neon-vb-v4/examples.json:1]: This is literally from our codec generator right? No manual modifications?

File: vendors/twtg/models/neon-vb_v4/model.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# Functional description of the product. Maximum 500 characters.
+description: "The NEON Vibration Sensor provides insight on vibrations (acceleration and velocity) of industrial equipment. Triggers can be set, based on which the sensors can send alerts."
+# Logo of the device
+logo: twtg-vb.png
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+# id: as defined under /profiles.
+# lorawanDeviceProfileID: as defined by LoRa Alliance in https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf.
+# The lorawanDeviceProfileID consists of 8 upper-case hexadecimal characters, 4 characters for VendorID + 4 characters for VendorProfileID. Please consider using single (') or double (") quotation.
+deviceProfileIds:
+    - id: twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA
```
Comment (martinichka) [inline vendors/twtg/models/neon-vb_v4/model.yaml:12]: What the `a` is for? twtg_RFGroup1_1.0.4**a**_RP2-1.0.3_classA

File: vendors/twtg/drivers/neon-vb-v4/examples.json (hunk 1 above)
Comment (aajn) [reply vendors/twtg/drivers/neon-vb-v4/examples.json:1]: Yes, I've taken the examples from the build output

File: vendors/twtg/models/neon-vb_v4/model.yaml (hunk 2 above)
Comment (aajn) [reply vendors/twtg/models/neon-vb_v4/model.yaml:12]: I do not know. I've copied the profiles from the templates:

image

Comment (martinichka) [reply vendors/twtg/models/neon-vb_v4/model.yaml:12]: Ok, lets ask them in the MR

**Pull request closed without merging**